pandas/tests/io/json/test_normalize.py

```diff
@@ -378,6 +397,31 @@ def test_json_normalize_errors(self):
                       ['general', 'trade_version']],
                 errors='raise')
 
+    def test_missing_meta(self, address_missing_data):
+        # GH25468: If metadata is nullable with errors set to ignore, the null
```

Comment:
Very minor but let's just shorten the comment to only reference the GitHub issue

Reply:
You can also just put what follows the "GH25468" part on a new line.  No need to delete it.